content/PGP/index.adoc

```diff
@@ -1,7 +1,10 @@
 == What is PGP? ==
 abbr:OpenPGP[Pretty Good Privacy] is an open standard for signing and encrypting. It enables RSA or ECC sign/encrypt operations using a private key stored on a smartcard (such as the YubiKey NEO), through common interfaces like PKCS#11.
 
-NOTE: PGP is not used for web authentication.
+NOTES:
```

Comment:
`NOTE` is an [admonition](https://asciidoctor.org/docs/asciidoc-writers-guide/#admonitions) in AsciiDoc. You probably want to define an admonition block instead:

```
[NOTE]
====
- The YubiKey does not support ECC PGP operations.
- PGP is not used for web authentication.
====
```